taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)
+
+
+class DriverUpdateForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=Driver.objects.all(),
+        widget=forms.CheckboxSelectMultiple()
+    )
+    fields = ("model", "manufacturer", "drivers")

```

Comment:
The `fields` attribute is incorrectly placed outside of the `Meta` class. It should be defined within the `Meta` class to specify which fields are included in the form.